pegasus/helpers/hourofcode_helpers.rb

```diff
@@ -149,9 +149,13 @@ def campaign_date(format)
   end
 end
 
+def all_unique_events
+  DB[:forms].where(kind: 'HocSignup2015').group(:name, :email).select(:name, :email, :processed_data, :data)
```

Comment:
@laurelfan you're right. This new query does a 'group by' instead of distinct and still selects all the needed data.